internal/manifest/version_edit_test.go

```diff
@@ -45,11 +45,12 @@ func checkRoundTrip(e0 VersionEdit) error {
 func TestVERoundTripAndAccumulate(t *testing.T) {
 	cmp := base.DefaultComparer.Compare
 	m1 := (&FileMetadata{
-		FileNum:        810,
-		Size:           8090,
-		CreationTime:   809060,
-		SmallestSeqNum: 9,
-		LargestSeqNum:  11,
+		FileNum:         810,
+		Size:            8090,
+		CreationTime:    809060,
+		SmallestSeqNum:  9,
+		LargestSeqNum:   11,
+		SyntheticSuffix: []byte("foo"),
```

Comment:
while you're here, could you add a synthetic prefix?

Reply:
Good idea, done. TFTR!